route/build/cpl/RtmMod.F90

```diff
@@ -739,10 +738,10 @@ SUBROUTINE get_river_export_data(NETOPO_in, RCHFLX_in, &
         if (present(offset)) ix=ix+offset
         ! stream volume is split into HRUs based on HRU's areal weight for contributory area
         if (update_vol) then
-          rtmCTL%volr(ix) = RCHFLX_in(iens,iRch)%ROUTE(idxIRF)%REACH_VOL(1)*NETOPO_in(iRch)%HRUWGT(iHru)/rtmCTL%area(ix)
+          rtmCTL%volr(ix) = RCHFLX_in(iens,iRch)%ROUTE(1)%REACH_VOL(1)*NETOPO_in(iRch)%HRUWGT(iHru)/rtmCTL%area(ix)
```

Comment:
As we talked about today, I'd like to have some explanation about why the index is hardcoded to "1". That could be in a comment for each line. But, better I think is to have an index called say something like "iRoute" that is fixed to 1, where you give the explanation that the route array is of size 1 for the cesm-coupled case. as only one routing type can be chosen. 

When I see a hardcoded "1" in an index I wonder why -- it's hardcoded to 1. So an explanation can be helpful.

Reply:
added `iRoute` parameter set to 1 and comments. then now using iRoute instead of hardcoded 1